lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -432,6 +432,7 @@
     </datatype>
     <datatype extension="velvet" type="galaxy.datatypes.assembly:Velvet" display_in_upload="true"/>
     <datatype extension="wig" type="galaxy.datatypes.interval:Wiggle" display_in_upload="true" description="The wiggle format is line-oriented.  Wiggle data is preceded by a track definition line, which adds a number of options for controlling the default display of this track." description_url="https://wiki.galaxyproject.org/Learn/Datatypes#Wig">
+    <datatype extension="agp" type="galaxy.datatypes.interval:AGP" description="The AGP format describes the assembly of a larger sequence object from smaller objects. The large object can be a contig, a scaffold (supercontig), or a chromosome. Each line (row) of the AGP file describes a different piece of the object, and has the column entries defined below." display_in_upload="true"/>
```

Comment:
This looks wrong. The converter belong to wig and not ago. Does the class agp exist?

Reply:
Yes, you are right. I didn't realize about the converters.